www/LICENSE

```diff
@@ -0,0 +1,14 @@
+The BSD Zero Clause License (0BSD)
```

Comment:
Is this the correct license for us to use?

Reply:
nope.